DISCLAIMER-STANDARD

```diff
@@ -0,0 +1,10 @@
+Apache Hudi (incubating) is an effort undergoing incubation
```

Comment:
should it be just named `DISCLAIMER` or does a doc explicity want it named `DISCLAIMER-STANDARD`

Reply:
+1 - please change to just DISCLAIMER.

Reply:
Done, please review again cc @vinothchandar